packages/trpc/server/routers/viewer/teams/removeMember.handler.ts

```diff
@@ -26,9 +27,15 @@ export const removeMemberHandler = async ({ ctx, input }: RemoveMemberOptions) =
     teamIds.map(async (teamId) => await isTeamAdmin(ctx.user.id, teamId))
   ).then((results) => results.every((result) => result));
 
-  const isOrgAdmin = ctx.user.profile?.organizationId
-    ? await isTeamAdmin(ctx.user.id, ctx.user.profile?.organizationId)
-    : false;
+  const isOrgAdmin = await Promise.all(
+    teamIds.map(async (teamId) => {
+      const userRepository = new UserRepository();
+      return await userRepository.isAdminOfTeamOrParentOrg({
+        userId: ctx.user.id,
+        teamId: teamId,
+      });
+    })
+  ).then((results) => results.some((result) => result));
```

Comment:
DevinAI, can we bulk check multiple teams in a single query? This feels like it can be a bottleneck.